### PR TITLE
applicant name change

### PR DESCRIPTION
**Pull request by KrishMunot**
Made the necessary changes

### CODE REVIEW

File: meta/labs/rejection-email.md [hunk 1]
```diff
@@ -11,4 +13,5 @@ Leaders community: [https://www.facebook.com/groups/1501083703514499]
 Introduce yourself by sharing one challenge you know you'll face with your
 club this year.
 
-Looking forward to watching the coding community grow at your school, Dave
+Looking forward to watching the coding community grow at your school
+Dave
```
Comment (MaxWofford) [inline meta/labs/rejection-email.md]: Markdown renders this as a single line still. Would you please put a newline between line 16 and 17?

File: meta/labs/rejection-email.md [hunk 2]
```diff
@@ -11,4 +13,5 @@ Leaders community: [https://www.facebook.com/groups/1501083703514499]
 Introduce yourself by sharing one challenge you know you'll face with your
 club this year.
 
-Looking forward to watching the coding community grow at your school, Dave
+Looking forward to watching the coding community grow at your school
```
Comment (MaxWofford) [inline meta/labs/rejection-email.md]: Was the comma removed on purpose?

**Comment by MaxWofford** [comment]
@KrishMunot Thanks for the contribution! Added a few comments.

File: meta/labs/rejection-email.md (hunk 2 above)
Comment (KrishMunot) [inline meta/labs/rejection-email.md]: Oops, not really. Will add it back.

File: meta/labs/rejection-email.md (hunk 1 above)
Comment (KrishMunot) [inline meta/labs/rejection-email.md]: Added a 'Dave' to the next line.

**Comment by KrishMunot** [comment]
Hey! I made the changes that you asked me to do @MaxWofford . Yet, I do not know why this automated script is not allowing me to submit a successful PR.
Kindly help me out. 
Thanks!

**Comment by MaxWofford** [comment]
@KrishMunot Will do!

**Comment by zachlatta** [comment]
Hey @KrishMunot! Thank you for the awesome fix.

**Pull request closed without merging**